src/jumpstart/SelectJumpstartRecipientButton.tsx

```diff
@@ -63,6 +64,7 @@ const styles = StyleSheet.create({
   body: {
     flexDirection: 'row',
     alignItems: 'center',
+    padding: Spacing.Regular16,
```

Comment:
the changes in this file is just moving containers around so that the android ripple on tap does not exceed the rounded green border